src/Logging/Logging.Console/src/ConsoleLoggerOptions.cs

```diff
@@ -9,12 +9,17 @@ public class ConsoleLoggerOptions
         public bool DisableColors { get; set; }
 
         /// <summary>
-        /// Gets or sets value indicating the minimum level of messaged that would get written to  <c>Console.Error</c>.
+        /// Gets or sets log message format. Defaults to <c>ConsoleLoggerFormat.Default</c>.
+        /// </summary>
+        public ConsoleLoggerFormat Format { get; set; }
```

Comment:
I see it defaults to `Default`, but to make it more clear I'd prefer to set it explictely. 
```suggestion
        public ConsoleLoggerFormat Format { get; set; } = ConsoleLoggerFormat.Default;
```

Reply:
Yeah, I'd agree. When the default is super-obvious (`int` being `0`, `bool` being `false`, etc.) I don't like initializing it, but here the default is only due to the `Default` value being first in the list.